DisGeNET.pm

```diff
@@ -0,0 +1,202 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+
+=cut
+
+=head1 NAME
+
+ DisGeNET
+
+=head1 SYNOPSIS
+
+ mv DisGeNET.pm ~/.vep/Plugins
+ ./vep -i variations.vcf --plugin DisGeNET,/path/to/disgenet/data.tsv.gz
+  ./vep -i variations.vcf --plugin DisGeNET,/path/to/disgenet/data.tsv.gz,1
+
+=head1 DESCRIPTION
+
+ This is a plugin for the Ensembl Variant Effect Predictor (VEP) that
+ adds Variant-Disease-PMID associations from the DisGeNET database.
+ It is available for GRCh38.
+
+ Please cite the DisGeNET publication alongside the VEP if you use this resource:
+ https://academic.oup.com/nar/article/48/D1/D845/5611674
+
+
+ Running options:
+ (Option 1) By default, this plugin includes three values = pmid:rsid:DisGeNET score.
+ (Option 2) It can be run with the option '1' to also include diseases/phenotypes
+ names reporting the Variant-pmid association.
+
+ Output:
+ The output includes three values: 
+  - pmid of the publication reporting the Variant-Disease association
+  - dbSNP variant Identifier
+  - DisGENET score for the Variant-Disease association
+  Example '20054638:rs10012:0.01'
+
+
+ The following steps are necessary before running this plugin:
+ This plugin uses file 'all_gene_disease_pmid_associations.tsv.gz'
+ File can be downloaded from: https://www.disgenet.org/downloads
+
+ gunzip all_gene_disease_pmid_associations.tsv.gz
+
+ awk '($1 ~ /^snpId/ || $2 ~ /NA/) {next} {print $0 | "sort -k2,2 -k3,3n"}' 
+ all_gene_disease_pmid_associations.tsv > all_variant_disease_pmid_associations_sorted.tsv
+
+ bgzip all_variant_disease_pmid_associations_sorted.tsv
+ tabix -s 2 -b 3 -e 3 all_variant_disease_pmid_associations_sorted.tsv.gz
+
+ The plugin can then be run as default (Option 1):
+ ./vep -i variations.vcf --plugin DisGeNET,/path/to/disgenet_file.tsv.gz
```

Comment:
The file that is created & indexed is called all_variant_disease_pmid_associations_sorted.tsv.gz

but the VEP command line is 
./vep -i variations.vcf --plugin DisGeNET,file=all_variant_disease_pmid_associations.tsv.gz

It would be better to use the name of the sorted, indexed file here.

Reply:
I didn't notice, it's fixed now.